### ISSUE TITLE
Support multiple migrations-directories

### ISSUE THREAD

**Issue by marcusrettig** [psalm/psalm-plugin-laravel#342]
**Is your feature request related to a problem? Please describe.**
The plugin only considers migrations in the `database/migrations` directory. If migrations are spread out in several directories, e.g. when using [Laravel Modules](https://nwidart.com/laravel-modules/v6/introduction), some Model stubs are generated without `@property` definitions. This results in many false positive `UndefinedMagicPropertyAssignment` errors.

**Describe the solution you'd like**
First, a warning message would be nice when a table can't be found ([here](https://github.com/psalm/psalm-plugin-laravel/blob/a91791e30e918bcc7dcc59a1307b1f63e705c78a/src/Fakes/FakeModelsCommand.php#L67)).

Second, a way to specify multiple directories for migrations would be nice. I'm not sure what would be the best way to do this.

**Describe alternatives you've considered**
Temporarily moving all migrations into `database/migrations` before running Psalm fixes the issue, but it is rather inconvenient.

**Additional context**
Our directory structure looks like this:

```
database/
    migrations/
modules/
    MobuleA/
        database/
            migrations/
    MobuleB/
        database/
            migrations/
```

**Comment by Ar-Monta** [comment]
Hello @marcusrettig 
To support migrating from a specific migration directory you can customize your module's service provider by overriding boot() function and adding this line of code:
`$this->loadMigrationsFrom("YOUR_SPECIFIC_MODULE_MIGRATIONS_DIRECTORY");`

**Comment by alies-dev** [comment]
hey @marcusrettig 

Would for your case be acceptable to provide this list of directories via XML config. Example (psalm.xml partial):
```xml
...
 <plugins>
        <pluginClass class="Psalm\LaravelPlugin\Plugin">
            <migrationsPath>database/migrations</migrationsPath>
            <migrationsPath>modules/ModuleA/database/migrations</migrationsPath>
            <migrationsPath>modules/ModuleB/database/migrations</migrationsPath>
        </pluginClass>
    </plugins>
...
```
?
